src/org/exist/xquery/functions/inspect/InspectModule.java

```diff
@@ -11,52 +11,54 @@
 
 public class InspectModule extends BasicFunction {
 
-    
-        public final static FunctionSignature FNS_INSPECT_MODULE = new FunctionSignature(
+
+    public final static FunctionSignature FNS_INSPECT_MODULE = new FunctionSignature(
             new QName("inspect-module", InspectionModule.NAMESPACE_URI, InspectionModule.PREFIX),
-            "Compiles a module from source (without importing it) and returns an XML fragment describing the " +
-            "module and the functions/variables contained in it.",
-            new SequenceType[] {
-                new FunctionParameterSequenceType("location", Type.ANY_URI, Cardinality.EXACTLY_ONE,
-                        "The location URI of the module to inspect"),
+            "Compiles a library module from source (without importing it) and returns an XML fragment describing the " +
+                    "module and the functions/variables contained in it.",
+            new SequenceType[]{
+                    new FunctionParameterSequenceType("location", Type.ANY_URI, Cardinality.EXACTLY_ONE,
+                            "The location URI of the module to inspect"),
             },
             new FunctionReturnSequenceType(Type.ELEMENT, Cardinality.ZERO_OR_ONE,
-                "An XML fragment describing the module and all functions contained in it.")
-        );
-        
-        public final static FunctionSignature FNS_INSPECT_MODULE_URI = new FunctionSignature(
+                    "An XML fragment describing the module and all functions contained in it.")
+    );
+
+    public final static FunctionSignature FNS_INSPECT_MODULE_URI = new FunctionSignature(
             new QName("inspect-module-uri", InspectionModule.NAMESPACE_URI, InspectionModule.PREFIX),
             "Returns an XML fragment describing the " +
-            "module identified by the given URI and the functions/variables contained in it.",
-            new SequenceType[] {
-                new FunctionParameterSequenceType("uri", Type.ANY_URI, Cardinality.EXACTLY_ONE,
-                    "The location URI of the module to inspect"),
+                    "library module identified by the given namespace URI and the functions/variables contained in it.",
+            new SequenceType[]{
+                    new FunctionParameterSequenceType("uri", Type.ANY_URI, Cardinality.EXACTLY_ONE,
```

Comment:
and confuse come in ...
`importModule(args[0].getStringValue(), null, null)` -> `importModule( String namespaceURI, String prefix, String location )`

and also you change description to point that it's `namespace URI`.